GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+On decisions regarding a release, the release manager(s) take this position.
+
+## Meetings
+
+There are 3 types of meetings that maintainers may participate in:
+
+- The annual General Assembly (GA),
+- The quarterly Virtual Maintainer Assembly (VMA), and
+- and the Weekly Coordinational Meeting.
+
+The GA is public and anyone who sees themselves as a member of the RIOT community can participate.
+Larger community steering decisions for the community are made during the GA, e.g., electing contact people for the Code of Conduct.
+The GA usually takes place during the [RIOT Summit](https://summit.riot-os.org/), the annual get-together of the RIOT community.
+The GA moderator usually is appointed by the organizers of the RIOT Summit.
+It is usually recorded and its notes will be published publicly in the RIOT forum.
+The agenda for the GA is collected before the assembly but may be bashed at the start of the meeting.
+
+The Virtual Maintainer Assembly (VMA) is a closed meeting among maintainers.
+The VMA appoints the release manager for upcoming releases and the moderator for the next VMA.
+Other maintenance decisions such as the fate of larger sections of code are discussed after these administrative tasks are done.
+The VMA usually takes place about a month after the latest release, usually in a virtual space, such as a video conference, the VMA moderator polls the maintainers for a sufficient date around the date of the upcoming release.
+The VMA may or may not co-incide with the GA, usually the VMA moderator decides, however, to merge these two meetings.
+In this case, VMA moderator and RIOT Summit organizers decide together on who is moderating the joint event (it may be the VMA moderator, it may be someone else).
+The agenda for the VMA is collected before the assembly but may be bashed at the start of the meeting.
+The notes of the VMA will be published publicly in the RIOT forum.
+The agenda for the VMA is collected before the assembly but may be bashed at the start of the meeting.
+
+The Weekly Coordinational Meeting is a closed meeting among maintainers.
+It usually serves as a small communal get-together of maintainers on a regular basis.
+Smaller maintenance decisions are made during these meetings, but also short term admistrative tasks are discussed.
+The Weekly Coordinational Meeting usually takes place every Friday at 10:00 in a virtual space, such as a video conference.
+A maintainer that feels responsible for it shares the link to the meeting as well as a proposed agenda, which may be amended by other maintainers, usually a day in advance.
+
+## Code of Conduct
+
+[Code of Conduct] violations by community members will be discussed and resolved on the conduct@riot-os.org list.
+If one of the appointees to that list (see GA notes for the members of that list) is involved in a Code of Conduct violation, two forum moderators from other organizations than the appointee take their place in the discussions.
```

Comment:
> see GA notes for the members of that list

Could we have a clearer pointer and/or directly a list of appointees here?

Reply:
Having an actual list of people in this document should be avoided. Since changing this document is supposed to be hard, it would add a lot of administrative overhead to update that.

Reply:
I see, so having _another_ place with a permanent link to check for the current appointees would be great.

Reply:
Maybe we should add them to https://www.riot-os.org/maintainers.html? @OlegHahm what do you think?

Reply:
Come to think of it... I am not sure this is a good idea. It gives the impression that only maintainers can be Code of Conduct contacts or that that role is somehow connected to be a maintainer in other ways. Since the list is not expected to be changing that much. Maybe we just should add them to [CODE_OF_CONDUCT.md](https://github.com/RIOT-OS/RIOT/blob/master/CODE_OF_CONDUCT.md)?

Reply:
See https://github.com/RIOT-OS/RIOT/wiki/CoC:-Reporting-Guidelines and https://github.com/RIOT-OS/RIOT/pull/21071 (waiting with updating this PR until either #21071 or we decide to stick to the wiki)